### ISSUE TITLE
Unigram performance varies greatly across models

### ISSUE THREAD

**Issue by redwyd** [THU-BPM/MarkLLM#42]
The F1 value of Unigram on opt-6.7b can reach 0.99, but the F1 value on gpt-j-6b is only 0.66. Is this normal?

**Comment by panly2003** [comment]
This is not normal. It's important to note that some models may have mismatched lengths between the len(tokenizer) and the logits vector dimensions during the forward pass. Make sure to check the dimensions of the logits vector at that time, and when initializing the transformers_config, set the vocab_size accordingly to match this size.
